### PR TITLE
Little optimization on getAdded/Removed activeobjects per player loop.

### PR DESCRIPTION
**Pull request by nerzhul**
Use std::queue instead of std::set, we don't need a so heavy container.
Don't convert position to int to convert it to float in the next function

### CODE REVIEW

**Comment by nerzhul** [comment]
@est31 if you get time to review this easy PR (some lines because of coding style)

File: src/environment.cpp [hunk 1]
```diff
@@ -1399,20 +1398,19 @@ void ServerEnvironment::getRemovedActiveObjects(v3s16 pos, s16 radius,
 		u16 id = *i;
 		ServerActiveObject *object = getActiveObject(id);
 
-		if(object == NULL){
-			infostream<<"ServerEnvironment::getRemovedActiveObjects():"
-					<<" object in current_objects is NULL"<<std::endl;
-			removed_objects.insert(id);
+		if(object == NULL) {
```
Comment (est31) [inline src/environment.cpp]: space

File: src/environment.cpp [hunk 2]
```diff
@@ -1399,20 +1398,19 @@ void ServerEnvironment::getRemovedActiveObjects(v3s16 pos, s16 radius,
...
+			infostream << "ServerEnvironment::getRemovedActiveObjects():"
+					<< " object in current_objects is NULL" << std::endl;
```
Comment (est31) [inline src/environment.cpp]: One tab only

File: src/environment.cpp [hunk 3]
```diff
@@ -1399,20 +1398,19 @@ void ServerEnvironment::getRemovedActiveObjects(v3s16 pos, s16 radius,
...
+			removed_objects.push(id);
 			continue;
 		}
 
-		if(object->m_removed || object->m_pending_deactivation)
-		{
-			removed_objects.insert(id);
+		if(object->m_removed || object->m_pending_deactivation) {
```
Comment (est31) [inline src/environment.cpp]: space

File: src/server.cpp [hunk 4]
```diff
@@ -697,27 +696,23 @@ void Server::AsyncRunStep(bool initial_step)
 				continue;
 
 			Player *player = m_env->getPlayer(client->peer_id);
-			if(player==NULL)
-			{
+			if(player == NULL) {
```
Comment (est31) [inline src/server.cpp]: space

File: src/server.cpp [hunk 5]
```diff
@@ -697,27 +696,23 @@ void Server::AsyncRunStep(bool initial_step)
...
 					client->m_known_objects, removed_objects);
-			m_env->getAddedActiveObjects(pos, radius, player_radius,
+			m_env->getAddedActiveObjects(player, radius, player_radius,
 					client->m_known_objects, added_objects);
 
 			// Ignore if nothing happened
-			if(removed_objects.empty() && added_objects.empty())
-			{
-				//infostream<<"active objects: none changed"<<std::endl;
+			if(removed_objects.empty() && added_objects.empty()) {
```
Comment (est31) [inline src/server.cpp]: space

**Comment by est31** [comment]
This is something for after the freeze.

**Comment by nerzhul** [comment]
Oh i was a little bit tired when fixing coding style, doing it soon

**Comment by est31** [comment]
@nerzhul do you have time now?

**Comment by est31** [comment]
Did the changes myself :-) 9c635f28ac2b103bf7196e47947bf64673c13393

**Pull request closed without merging**